socorro/external/es/new_crash_source.py

```diff
@@ -92,11 +89,11 @@ def new_crashes(self, date, product, versions):
                 scroll='2m',  # keep the "scroll" connection open for 2 minutes
                 index=es_index,
                 doc_type=es_doctype,
-                fields=['crash_id'],
                 query=query,
+                _source=False,
             )
             for hit in res:
-                crash_ids.append(hit['fields']['crash_id'][0])
+                crash_ids.append(hit['_id'])
```

Comment:
It's always been the case, and I used this opportunity to make this a little more efficient by not needing any field to be returned.